src/popover/__tests__/popover-focus-loop.scenario.js

```diff
@@ -32,7 +32,7 @@ export function Scenario() {
         </div>
       )}
     >
-      <button>click</button>
```

Comment:
The trigger for this popover is set to "hover", so this button should be named to reflect that.